### DISCUSSION TITLE
different between pycuda.driver.mem_get_info and nvidia-smi

### DISCUSSION THREAD

**Discussion by BlueWinters** [inducer/pycuda#323] [Q&A]
the output about gpu memory usage between pycuda.driver.mem_get_info() and nvidia-smi is different.
what is the actual meaning for the two?

> for pycuda.driver.mem_get_info
[https://documen.tician.de/pycuda/driver.html#pycuda.driver.mem_get_info](url)

**Comment by inducer** [comment]
It might be a good idea to check the Nvidia docs for both, as pycuda merely passes on what it gets from the CUDA driver API. Could you do that and post links?
